internal/controller/grafanaorganization_controller.go

```diff
@@ -221,3 +254,12 @@ func (r GrafanaOrganizationReconciler) reconcileDelete(ctx context.Context, graf
 
 	return nil
 }
+
+func updateGrafanaOrganizationInfoMetric(organizationName string, displayName string, orgID int64, status string) {
+	metrics.GrafanaOrganizationInfo.WithLabelValues(
+		organizationName,
+		displayName,
+		fmt.Sprintf("%d", orgID),
+		status,
+	).Set(1)
```

Comment:
@QuantumEnigmaa I think this should not be set to 1 all the time right? Or if it is, then the other status should be set to 0 right?

Reply:
Oh yes you're right, I was confused with the _status metrics in ksm